src/twisted/conch/test/test_conch.py

```diff
@@ -118,9 +122,10 @@ class ConchTestOpenSSHProcess(protocol.ProcessProtocol):
     fired twice. Fires when the process is terminated.
     """
```

Comment:
I tried to document the new functionality of this test helper

```suggestion
    fired twice. Fires when the process is terminated.
    
    @ivar expectedExitCode: If the process exit code is not C{expectedExitCode}
    the set C{deferred} will by triggerd with a failure.
    """
```